toolkits/global/packages/global-forms/scss/00-tokens/_default.tokens.scss

```diff
@@ -0,0 +1,46 @@
+// Generated on 30/06/2022, 08:09:54
```

Comment:
Am I right in thinking lots of these variables are not being used in the component styling? From what I can tell these are defined but not used:

$global-forms-checkbox-checked-icon-fill
$global-forms-checkbox-checked-icon-path
$global-forms-checkbox-checked-icon-stroke
$global-forms-checkbox-size
$global-forms-heading-font-family
$global-forms-heading-font-size
$global-forms-heading-font-weight
$global-forms-indent-border-radius
$global-forms-indent-border-width
$global-forms-input-border-color-error
$global-forms-input-font-color
$global-forms-label-font-size
$global-forms-legend-font-family
$global-forms-legend-font-size
$global-forms-legend-font-weight
$global-forms-message-color-resting
$global-forms-message-font-size
$global-forms-message-font-weight
$global-forms-message-icon-color-error
$global-forms-message-icon-color-resting
$global-forms-radio-carat-color
$global-forms-radio-carat-size
$global-forms-radio-carat-spacing
$global-forms-select-icon-stroke

Perhaps I'm missing something? 😄

Reply:
@benjclark No, you're quite right. @StuRobson and I need to audit what should be used and what's redundant.

Reply:
it's also a case of me being explicit in trying to define everything … and @Heydon (perhaps rightly (😛), considering the overall architecture of the CSS) relying on the cascade